spec/plans/report.fmf

```diff
@@ -202,33 +203,7 @@ description:
 /file:
     description: |
 
-        Save the report into a ``report.yaml`` file with the
-        following format:
-
-        .. code-block:: yaml
-
-            result: OVERALL_RESULT
-            plans:
-                /plan/one:
-                    result: PLAN_RESULT
-                    tests:
-                        /test/one:
-                            result: TEST_RESULT
-                            log:
-                              - LOG_PATH
-
-                        /test/two:
-                            result: TEST_RESULT
-                            log:
-                                - LOG_PATH
-                                - LOG_PATH
-                                - LOG_PATH
-                /plan/two:
-                    result: PLAN_RESULT
-                        /test/one:
-                            result: TEST_RESULT
-                            log:
-                              - LOG_PATH
+        Save the report into a ``report.yaml`` file.
 
         Where ``OVERALL_RESULT`` is the overall result of all plan
```

Comment:
```suggestion
        The ``OVERALL_RESULT`` is the overall result of all plan
```

Moving the example down sounds ok, but `Where` is referencing to something which has been already mentioned above. What about using `The` instead? Same for the other values below.

Reply:
@psss done!

Reply:
Thanks!